Sources/UserModuleKit/Interfaces/Account+Interface.swift

```diff
@@ -22,16 +22,16 @@ public protocol UserAccountInterface {
     ) async throws -> User.Account.Detail
 
     func get(
```

Comment:
It should be renamed to "require" because of Detail return is not optional (not Detail?)